extensions/shared/external-media/editor.scss

```diff
@@ -5,7 +5,7 @@
 
 $grid-size: 8px;
 
-@keyframes loading-fade {
+@keyframes external-media-loading-fade {
```

Comment:
Should keyframes here also be prefixed with jetpack?

Reply:
Fixed in e74fd89b071a761da02017edd0ec214caf9c872a